### DISCUSSION TITLE
Thumbnails !

### DISCUSSION THREAD

**Discussion by yourmomlover** [EchterAlsFake/EPorner_API#2] [Ideas]
after reading the documentation , it seems it would be possible to retrieve the thumbnails  , im trying to return it by thumbnail_url but it dosen't seem to work , could u explain the thumbnail structure  what it returns its & how i could implement it

**Comment by EchterAlsFake** [comment]
I'll work on the Eporner and HQporner thumbnail tonight or tomorrow. They are currently a bit broken. I need to update all the APIs in general xD

**Reply by yourmomlover** [reply]
ah alr lmao , btw are u gonna do sex.com/pins scraper too?

**Reply by EchterAlsFake** [reply]
Yes.

**Reply by EchterAlsFake** [reply]
Thumbnails should be fixed now. It now returns a single thumbnail which is the one you see, before the video starts.

Thanks :)

**Comment by yourmomlover** [comment]
> Yes.
 sheesh thanks



> Thumbnails should be fixed now. It now returns a single thumbnail which is the one you see, before the video starts.
> 
> Thanks :)

how do i get the thumbnails , i thought it would be the same as the Hq one `video.get_thumbnails() ` but i get this error `'Video' object has no attribute 'get_thumbnails'`

**Reply by EchterAlsFake** [reply]
Uhm no, it's video.thumbnail

Don't you use an IDE for coding xD

**Reply by yourmomlover** [reply]
i use vscode  , when i downloaded it there were some issues with v env so i never got it lmao
